src/controllers/update/remove.is.entity.registered.owner.controller.ts

```diff
@@ -19,7 +26,15 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
 export const post = (req: Request, res: Response, next: NextFunction) => {
   try {
     logger.debugRequest(req, `POST ${config.REMOVE_IS_ENTITY_REGISTERED_OWNER_PAGE}`);
-    if (req.body["owner_disposed"] === 'no') {
+    const isListedAsPropertyOwner = req.body[IsListedAsPropertyOwnerKey];
+
+    const appData: ApplicationData = getApplicationData(req.session);
+    const remove = getRemove(appData);
+    remove[IsListedAsPropertyOwnerKey] = isListedAsPropertyOwner;
+
+    setApplicationData(req.session, remove, RemoveKey);
+
+    if (isListedAsPropertyOwner === '0') {
```

Comment:
yes, could introduce new constants and start using them.

Reply:
> Would be nice to have a couple of string constants defined for these '0' and '1' values (they appear in a lot of places and not that readable), e.g. `OPTION_NO` and `OPTION_YES`. Too late?

I agree. It would make it more readable and was something I raised in an earlier review.

Reply:
👍

Reply:
update to use config values for '1' and '0', went with OPTION_YES , OPTION_NO

Reply:
missed a unit test, i'll update it

Reply:
updated